.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,13 @@
+<!--
+Welcome!
+
+Help us stay on top of issues by submitting them to the right repository. For
+issues related to:
+* ... building images: https://github.com/jupyterhub/binder/new
+* ... the https://mybinder.org site not working or misbehaving: https://github.com/jupyterhub/mybinder.org-deploy/issues/new
```

Comment:
I think we should be more specific than this. E.g., `mybinder.org` downtime or inaccessible pages. Otherwise I feel like we'd get people with a build error who think this means that mybinder.org "isn't working"

Reply:
Adapted your language.